service.subtitles.napiprojekt/service.py

```diff
@@ -0,0 +1,199 @@
+# -*- coding: utf-8 -*- 
+
+from os import path
+from sys import path as sys_path, argv
+from urllib.parse import unquote, quote_plus
+from unicodedata import normalize
+from hashlib import md5
+
+import xbmc
+import xbmcvfs
+import xbmcaddon
+import xbmcgui
+import xbmcplugin
+
+__addon__ = xbmcaddon.Addon()
+__scriptid__ = __addon__.getAddonInfo('id')
+__scriptname__ = __addon__.getAddonInfo('name')
+__version__ = __addon__.getAddonInfo('version')
+__language__ = __addon__.getLocalizedString
+
+__cwd__ = xbmcvfs.translatePath(__addon__.getAddonInfo('path'))
+__profile__ = xbmcvfs.translatePath(__addon__.getAddonInfo('profile'))
+__resource__ = xbmcvfs.translatePath(path.join(__cwd__, 'resources', 'lib'))
+__temp__ = xbmcvfs.translatePath(path.join(__profile__, 'temp', ''))
+
+sys_path.append(__resource__)
```

Comment:
This is still here...

Reply:
what is the issue with it?

Reply:
It's an ugly hack and a bad practice, probably something you got from some legacy addon. Other devs get inspiration from the addons in repo so I prefer not to have addons doing it in the repo.
Plus, for some addon types kodi manipulates the sys.path itself (e.g. in context menu type addons, a listitem object is added to the sys.path), so addons should avoid messing with it.

Use:

```python
from resources.lib import NapiProjekt

NapiProjekt.whatever_method()
```

Reply:
I've tried it, it doesn't works :|